jenkins-scripts/dsl/_configs_/HelperRetryFailures.groovy

```diff
@@ -0,0 +1,62 @@
+package _configs_
+
+import javaposse.jobdsl.dsl.Job
+
+class HelperRetryFailures {
+
+  /**@
+    * Create method for Nagiator Publisher retry plugin
+    * @param job - Job to add the plugin to
+    * @param args - Map of arguments to pass to the plugin
+    * @param args.regexpForRerun - Regular expression to match against the console output
+    * @param args.checkRegexp - Whether to check the regular expression
+    * @param args.maxSchedule - Maximum number of times to retry the build
+    * @param args.delay - Delay in seconds to wait before retrying the build
+    */
+  static void create(Job job, Map args) {
+    job.with {
+      publishers {
+        configure { project ->
+          def old_regexpForRerun = (project / publishers / 'com.chikli.hudson.plugin.naginator.NaginatorPublisher' / regexpForRerun).text()
```

Comment:
The code is a bit ugly with all the long selectors. Maybe we can use something like:
```groovy
          def naginator_root = project / publishers / 'com.chikli.hudson.plugin.naginator.NaginatorPublisher'
          
          def old_regexpForRerun = (naginator_root  / regexpForRerun).text()
 ```

And apply the root for the rest of the code.